.github/workflows/run_unit_tests.yml

```diff
@@ -76,3 +76,36 @@ jobs:
     - name: Print unit test logs
       if: always()
       run: cat builddir/meson-logs/testlog.txt
+
+  windows:
+    name: Build and run tests on Windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+        cc: [clang]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install build and test dependencies
+      # Pin to Meson 0.60.1 for
+      # https://github.com/mesonbuild/meson/issues/10022
+      run: pip install meson==0.60.1
```

Comment:
Without this pin, I get:

```
subprojects\check-0.15.2\tests\libstaticcheck.a : fatal error LNK1107: invalid or corrupt file: cannot read at 0x2452
```

It appears from https://github.com/mesonbuild/meson/issues/10022 that this was fixed in 0.60.1 but the fix wasn't forward-ported to 0.61+.

cc @eli-schwartz

Reply:
Hi, another known workaround is to set both `clang++` and `lld-link` to `CXX` and `CXX_LLD` environment variables so you don´t have issues with the link issue while using a newer meson version like 1.0.0, thanks for the notice on this issue I will add it on the list of known workarounds

Reply:
This is a C project, so `CXX` and `CXX_LD` are ignored.  Setting `CC` to a C++ compiler fails at setup time, but setting only `CC_LD` works.  PR in https://github.com/hackermd/libdicom/pull/29; thanks for the pointer.

Reply:
right, mb, in that case as you mentioned `cc` should be `clang` but only `CC_LD` is required as you mentioned